### PR TITLE
Fix: Backend CI Build

### PR DESCRIPTION
**Pull request by stephane-segning**
Aim is to fix the CI pipeline.

- Removed the mvn from the Dockerfile 
- Put it inside the github workflow file

### CODE REVIEW

File: power-pay-backend/Dockerfile
Comment (mbunwe-victor) [inline power-pay-backend/Dockerfile:1]: The mvn has been removed as required

File: .github/workflows/build-service.yml [hunk 1]
```diff
@@ -10,55 +10,66 @@ on:
...
+      - name: Clone the project
+        uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: '17'
       - name: Maven build
-        run: cd power-pay-backend && mvn clean package -q
-      
+        run: mvn clean package -q
```
Comment (mbunwe-victor) [inline .github/workflows/build-service.yml:33]: here i can see mvn has been added in to the workflows file as required.

**Review by mbunwe-victor — APPROVED** [review]
The expected changes are well implemented  sir @stephane-segning

File: .github/workflows/build-service.yml (hunk 1 above)
Comment (stephane-segning) [reply .github/workflows/build-service.yml:33]: Yes, because the working dir was specified

**Review by kouamschekina — APPROVED** [review]

**Review by chendiblessing — APPROVED** [review]

**Pull request merged**